ONSdigital/rm-case-service
# Updated case events by case id endpoint to take optional category

# Motivation and Context
Rops users want to be able to get the timestamp for a completed `seft` or `eq`. We have decided for the first iteration to only return the first completed timestamp for SEFT. 

# What has changed
Updated endpoint to take optional category and return case events based on category, along with said created date stamp. 

# How to test?
Run in docker dev and hit endpoint to see if matched case events to categories and case id's are returned. 

Wanted categories:
`OFFLINE_RESPONSE_PROCESSED`
`COMPLETED_BY_PHONE`
`SUCCESSFUL_RESPONSE_UPLOAD`

Note this end point will return all categories sent if in DB. 

# Links
[TRELLO CARD](https://trello.com/c/sNa7NP3W/462-create-endpoint-in-case-service-to-expose-completion-details-incl-timestamp-m)
